removed mentioning of 'default theme' in theming

chludwig opened the pull request:
Since v8 there is no 'default' theme, but an exaple theme.

carlaschroder commented:
:+1: 
@MorrisJobke, does this look correct?

carlaschroder commented:
Thank you @chludwig!

MorrisJobke commented:
@carlaschroder @chludwig The problem here is, that the default theme is loaded always and therefore you can start right away with changing the theme files. With this documentation we need to move the very last paragraph (about the theme name and the config option) ti the beginning - otherwise it won't work.

chludwig commented:
@MorrisJobke I moved and changed the last paragraph. And what do you mean by ``default`` theme? There is no subfolder ``default`` within themes. Default means in your case ``core`` and ``settings``?

MorrisJobke commented:
> @MorrisJobke I moved and changed the last paragraph. And what do you mean by default theme? There is no subfolder default within themes. Default means in your case core and settings?

The default theme name is `default`. So if you create a folder `themes/default` and put your changed files in there it will get automatically loaded by owncloud without setting anything in the config.php. Therefore this manual uses it, because you just need to drop your changed files into the theme named `default` and it will be loaded always ;)

I know it's confusing.

MorrisJobke commented on developer_manual/core/theming.rst:
```diff
@@ -19,50 +19,72 @@ With this facts you can easily determine, where the following object-related att
+in your template, you must merge the changes that where made within the update
+with the ones you did in your template.
+
+But this is unlikely at least ownCloud aims to give further information in this case.
```
Somehow the paragraph about the `theme` setting in `config.php` is missing

chludwig commented:
@MorrisJobke this was already mentioned a paragraph before. I made it more prominent.

carlaschroder commented:
This needs reviewers and corrections. Not good to leave contributor PRs lying around
Is this thumbs up, down, or sideways? :)

@PVince81 @MorrisJobke @chludwig

carlaschroder commented:
Ahem. Ok then, this looks good to me. Thank you @chludwig

carlaschroder commented:
stable9: https://github.com/owncloud/documentation/commit/73ddff3c04e7cc574efda26d6cb48df362e8171b

Does this apply to older ownCloud versions?

MorrisJobke commented:
> Does this apply to older ownCloud versions?

Let's leave it as it is.
